src/cowboy_swagger_handler.erl

```diff
@@ -33,19 +33,19 @@ trails(Options) ->
             _ ->
                 filename:join(cowboy_swagger_priv(), "swagger")
         end,
+    Path = application:get_env(cowboy_swagger, path, "/api-docs"),
     Redirect =
-        trails:trail("/api-docs",
+        trails:trail(Path,
                      cowboy_swagger_redirect_handler,
-                     {file, StaticFiles ++ "/index.html"},
+                     {file, Path ++ "/index.html"},
```

Comment:
I seem to not be able to follow this change. Would you elaborate, please?

Reply:
It looks like here need use `StaticFiles` as it was before, because only path should be dynamic - maybe was added by mistake?